aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,366 @@
+
+import aepp
+from aepp import flowservice
+from aepp import destinationinstanceservice
+import time
+import logging
+from typing import Union
+from aepp.utils import Utils
+from .configs import ConnectObject
+from tenacity import Retrying
+from tenacity import retry_if_result
+from tenacity import wait_fixed
+from tenacity import stop_after_attempt
+from dataclasses import dataclass,asdict,field
+
+@dataclass
+class Flow:
+    name: str = None
+    flowSpec: dict = field(default_factory=dict)
+    sourceConnectionIds: list = field(default_factory=list)
+    targetConnectionIds: list= field(default_factory=list)
+    scheduleParams: dict = field(default_factory=dict)
+    transformations: list = field(default_factory=list)
+
+    def __init__(self,
+                 name: str,
+                 flow_spec: dict,
+                 source_connection_id: str,
+                 target_connection_id: str,
+                 schedule_params: dict,
+                 transformations: dict
+                 ):
+        self.name = name
+        self.sourceConnectionIds = [source_connection_id]
+        self.targetConnectionIds = [target_connection_id]
+        self.flowSpec = flow_spec
+        self.scheduleParams = schedule_params
+        self.transformations = transformations
+
+class ExportDatasetToCloudStorage:
+    """
+    A class for exporting dataset to cloud storage functionality
+    Attributes
+    ----------
+    flow_conn : module
+        flow service connection
+    dis_conn : module
+        destination instance service connection
+    username : str
+        the log in for the os
+    logger: module
+        class logger
+    """
+
+    flow_conn = None
+    dis_conn = None
+    username = None
+    logger = None
+    DLZ_CONNECTION_SPEC_ID = "10440537-2a7b-4583-ac39-ed38d4b848e8"
+    DLZ_FLOW_SEPC_ID = "cd2fc47e-e838-4f38-a581-8fff2f99b63a"
+    def __init__(
+            self,
+            username: str,
+            config: Union[dict,ConnectObject] = aepp.config.config_object,
+            header: dict = aepp.config.header
+    ):
+        """
+        initialize the Export Dataset to CloudStorage instance.
+        Arguments:
+            username : REQUIRED: username that will be used in cmle entity names
+            config : OPTIONAL : config object in the config module.
+            header : OPTIONAL : header object  in the config module.
+        """
+
+        self.username = username
+        self.flow_conn = flowservice.FlowService(config=config, header=header)
+        self.dis_conn = destinationinstanceservice.DestinationInstanceService(config=config, header=header)
+        self.logger = logging.getLogger(f"{__name__}")
+        self.logger.setLevel(logging.INFO)
+        #Create a stream-based handler that writes the log entries into the standard output stream
+        handler = logging.StreamHandler()
+        #Create a formatter for the logs
+        formatter = logging.Formatter('%(created)f:%(module)s:%(message)s')
```

Comment:
The rest of the library seems to be using a `loggingObject` and use that to create the formatter, can we use the same for consistency?